datalake/src/main/java/com/sequenceiq/datalake/flow/detach/event/SdxDetachFailedEvent.java

```diff
@@ -15,6 +15,6 @@ public static SdxDetachFailedEvent from(SdxEvent event, Exception exception) {
 
     @Override
     public String selector() {
-        return "SdxStartDetachEvent";
+        return "SdxDetachFailedEvent";
```

Comment:
this method shouldn't be overwritten at all. `SdxEvent` has a default selector implementation which is just fine

Reply:
I'll look into this, I was just sticking to the logic already present. Thank you for letting me know about this.

Reply:
Done.